server/src/test/scala/org/http4s/server/staticcontent/FileServiceSuite.scala

```diff
@@ -30,8 +30,9 @@ import java.io.File
 import java.nio.file._
 
 class FileServiceSuite extends Http4sSuite with StaticContentShared {
-  val defaultSystemPath = org.http4s.server.test.BuildInfo.test_resourceDirectory.getAbsolutePath
-  val routes = fileService(
+  private val defaultSystemPath =
+    org.http4s.server.test.BuildInfo.test_resourceDirectory.getAbsolutePath
+  val routes: HttpRoutes[IO] = fileService(
```

Comment:
This finding blew my mind. If remove this type annotation then tests are passed well. Or I miss something stupid obvious, or it's some magic away from Hogwarts.

Reply:
I'm still confused what was found.

Reply:
I reverted changes https://github.com/http4s/http4s/pull/5822/commits/a3cba5fa108ae68ff29d83e156b6d165a7e85c66 that were causing the test to fail.